src/main/java/Duke.java

```diff
@@ -1,10 +1,51 @@
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
 public class Duke {
     public static void main(String[] args) {
+        showWelcomeMessage();
+        handleUserInput();
+    }
+
+    public static void showWelcomeMessage(){
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        System.out.println("____________________________________________________________\n" +
+                " Hello! I'm Duke\n" + "What can I do for you?\n" +
+                "____________________________________________________________\n" );
+    }
+
+    public static void handleUserInput(){
+        Scanner in = new Scanner(System.in);
+        String inputText = "";
+        List<String> inputs = new ArrayList<>();
+        boolean isRunning = true;
+        while(isRunning){
+            inputText = in.nextLine();
+            if(inputText.equals("bye")) {
+                isRunning = false;
+            }
+            else if (inputText.equals("list")) {
+                System.out.println("____________________________________________________________");
+                for(int i=0;i< inputs.size();i++){
+                    System.out.println("\t"+(i+1)+". "+ inputs.get(i) );
+                }
+                System.out.println("____________________________________________________________");
```

Comment:
this line makes the code look long and messy, maybe can be replaced with a helper function defined above